### PR TITLE
Add com.abagames.rRootage

### PR DESCRIPTION
**Pull request by joaquimrocha**
It has a little more than just the manifest as the instructions usually indicate, but there were a number of things I needed to host for easier access (a few patches by Debian, screenshots which were take by me, etc.).

### CODE REVIEW

File: com.abagames.rrotage.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
+  "app-id": "com.abagames.rRootage",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
```
Comment (nedrichards) [inline com.abagames.rrotage.json]: Have you tried this against 18.08? 1.6 is out of security support and I'd encourage trying to use the more modern version if possible.

File: com.abagames.rrotage.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  "sdk": "org.freedesktop.Sdk",
+  "command": "rrootage",
+  "rename-desktop-file": "rrootage.desktop",
+  "rename-icon": "rrootage",
+  "rename-appdata-file": "rrootage.appdata.xml",
+  "finish-args": [
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11"
```
Comment (nedrichards) [inline com.abagames.rrotage.json]: add --share=ipc if you add socket=x11 http://docs.flatpak.org/en/latest/sandbox-permissions.html

**Comment by nedrichards** [comment]
bot, build com.abagames.rRootage

**Comment by nedrichards** [comment]
I do love rRootage, it'd be great to see it available.

**Comment by nedrichards** [comment]
Build fails because the JSON file name doesn't match the app-id.

**Comment by joaquimrocha** [comment]
I will address all these tonight. Thanks @nedrichards .

**Comment by joaquimrocha** [comment]
bot, build com.abagames.rRootage

**Comment by joaquimrocha** [comment]
@nedrichards Hopefully I've addressed all your comments.

**Comment by joaquimrocha** [comment]
The icon was missing (forgot to include it from the original repo I had created).

**Comment by joaquimrocha** [comment]
bot, build com.abagames.rRootage

**Comment by nedrichards** [comment]
bot, build com.abagames.rRootage

**Comment by joaquimrocha** [comment]
I hadn't added the release tags because they're from 2003, and g-s shows some info about an app "not having recent updates" (I am not sure the same is displayed for games though).I hadn't added the release tags because they're from 2003, and g-s shows some info about an app "not having recent updates" (I am not sure the same is displayed for games though).
In any case, I have added that info. Thanks again for the review @nedrichards .

**Comment by joaquimrocha** [comment]
I have forgotten to update the runtime version (1.6 -> 18.08). There you go.

**Comment by nedrichards** [comment]
bot, build com.abagames.rRootage

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.abagames.rRootage

**Pull request closed without merging**